emop-maven-plugin/src/main/java/edu/cornell/ImpactedClassMojo.java

```diff
@@ -15,9 +16,22 @@ public class ImpactedClassMojo extends ImpactedMojo {
     @Parameter(property = "updateChecksums", defaultValue = "true")
     private boolean updateChecksums;
 
+    /**
+     * Parameter to determine which closure to use for impacted classes.
+     * TRANS_OF_INVERSE_TRANS = p1
+     * TRANS_AND_INVERSE_TRANS = p2
+     * TRANS = p3
+     */
+    @Parameter(
+            property = "closureOption",
+            defaultValue = "TRANS"
+    )
+    private TransitiveClosureOptions closureOption;
+
     public void execute() throws MojoExecutionException {
         setUpdateImpactedChecksums(updateChecksums);
         setTrackNewClasses(true);
+        setTransitiveClosureOption(closureOption);
```

Comment:
I am assuming that his method is in STARTS, and that is why its implementation is not in this diff?

Reply:
Yes this method is with the other setters in STARTS